checkov/terraform/checks/resource/ncp/AccessControlGroupOutboundRule.py

```diff
@@ -0,0 +1,25 @@
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+from checkov.common.models.enums import CheckResult, CheckCategories
+
+
+class AccessControlGroupOutboundRule(BaseResourceCheck):
+    def __init__(self):
+        name = "An outbound security group rule allows traffic to /0"
+        id = "CKV_NCP_3"
+        supported_resources = ['ncloud_access_control_group_rule']
+
+        categories = [CheckCategories.NETWORKING]
+        guideline = "You should restrict access to IP addresses or ranges that are explicitly required where possible."
```

Comment:
```suggestion
```
guidelines for included checks are added separately by us, so no need to do it here. just add some explanation in the description of the PR 🙂  and we will use it when we create the docs for it